docs/docs/external-volumes.md

```diff
@@ -172,15 +172,20 @@ The default implicit volume size is 16 GB. If you are using the Mesos containeri
 
 ### Potential Pitfalls
 
+*   If one or more external volumes are declared for a Marathon app, and the Docker image specification includes one or more `VOLUME` entries, Docker may create anonymous external volumes. This is default Docker behavior with respect to volume management when the `--volume-driver` flag is passed to `docker run`. Follow these steps to prevent Docker from creating anonymous volumes:
+  *  `docker inspect` the app's Docker image before running the app in Marathon and make a note of the `VOLUME` entries in the specification.
+  *   declare non-external, host-volume mounts in your app's container specification for each `VOLUME` entry that should not map to an anonymous external volume.
```

Comment:
Probably worth a code snippet. I'll put something together

On Mon, Jul 25, 2016 at 1:56 PM, Suzanne Scala <notifications@github.com>
wrote:

> In docs/docs/external-volumes.md
> <https://github.com/mesosphere/marathon/pull/4131#discussion_r72112030>:
>
> > @@ -172,15 +172,20 @@ The default implicit volume size is 16 GB. If you are using the Mesos containeri
> >
> >  ### Potential Pitfalls
> >
> > +*   If one or more external volumes are declared for a Marathon app, and the Docker image specification includes one or more `VOLUME` entries, Docker may create anonymous external volumes. This is default Docker behavior with respect to volume management when the `--volume-driver` flag is passed to `docker run`. Follow these steps to prevent Docker from creating anonymous volumes:
> > +  *  `docker inspect` the app's Docker image before running the app in Marathon and make a note of the `VOLUME` entries in the specification.
> > +  *   declare non-external, host-volume mounts in your app's container specification for each `VOLUME` entry that should not map to an anonymous external volume.
>
> Is it worth have a code snippet here?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4131/files/fe790975b35e9e54a4d52b6d45b89d79a87f63ca#r72112030>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLON0O8RLgsFb4thOxvAdSfptKplHks5qZPjFgaJpZM4JTHxm>
> .
>